docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+    "    s = pd.DataFrame(\n",
+    "        np.vstack(\n",
+    "            [\n",
+    "                generate_noisy_labels(true_labels_train, noise_matrix_better)\n",
+    "                if i < num_annotators - 5\n",
+    "                else generate_noisy_labels(true_labels_train, noise_matrix_worse)\n",
+    "                for i in range(num_annotators)\n",
+    "            ]\n",
+    "        ).transpose()\n",
+    "    )\n",
+    "\n",
+    "    # Each annotator only labels approximately 10% of the dataset\n",
+    "    # (unlabeled points represented with NaN)\n",
+    "    s = s.apply(lambda x: x.mask(np.random.random(n) < 0.9))\n",
+    "    s.dropna(axis=1, how=\"all\", inplace=True)\n",
+    "\n",
+    "    row_NA_check = pd.notna(s).any(axis=1)\n",
+    "\n",
+    "    return {\n",
+    "        \"X_train\": X_train[row_NA_check],\n",
+    "        \"true_labels_train\": true_labels_train[row_NA_check],\n",
+    "        \"multiannotator_labels\": s[row_NA_check].reset_index(drop=True),\n",
+    "    }\n",
+    "\n",
+    "data_dict = make_data()\n",
+    "\n",
+    "X = data_dict[\"X_train\"]\n",
+    "multiannotator_labels = data_dict[\"multiannotator_labels\"]\n",
+    "true_labels = data_dict[\"true_labels_train\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e95d31b2",
+   "metadata": {},
+   "source": [
+    "For this tutorial we are using a toy dataset that has 50 annotators and 160 examples. There are three possible classes, `0`, `1` and `2`.\n",
+    "\n",
+    "Each annotator annotates approximately 10% of the examples. We also synthetically made the last 5 annotators in our toy dataset have much noisier labels than the rest of the annotators."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "946726ad",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Bringing Your Own Data (BYOD)?\n",
+    "\n",
+    "You can easily replace the above with your own multiannotator dataset, and continue with the rest of the tutorial.\n",
+    " \n",
+    "`multiannotator_labels` should be a numpy array or pandas DataFrame with each column representing an annotator and each row representing an example. Your classes (and entries of `multiannotator_labels`) should be represented as integer indices 0, 1, ..., num_classes - 1, where examples that are not annotated by a particular annotator are represented using `np.nan`.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51335def",
+   "metadata": {},
+   "source": [
+    "## 3. Get majority vote label and compute out-of-sample predicted probabilites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1857cc7",
+   "metadata": {},
+   "source": [
+    "Before training our machine learning model, we must first obtain the consensus labels from the annotators that labeled the data. The simplest way to obtain an initial set of consensus labels is to select it using majority vote."
```

Comment:
can we add hyperlink here for: majority vote
to a stable website with clear definition/tutorial of what this is in the crowdsourcing context?